lib/helper/FileSystem.js

```diff
@@ -136,7 +170,34 @@ class FileSystem extends Helper {
 function getFileContents(file, encoding) {
   if (!file) assert.fail('No files were opened, please use seeFile action');
   encoding = encoding || 'utf8';
-  return fs.readFileSync(file, 'utf8');
+  return fs.readFileSync(file, encoding);
 }
 
 module.exports = FileSystem;
+
+function checkExistsWithTimeout(filePath, timeout) {
```

Comment:
Maybe checkFileExists, the timeout is an additional option we may eliminate it in function name.

Reply:
Moreover, "check" is absolutely meaningless name for function, especially for assert.
`ifFileExists` is better.